testing/openjfx11/APKBUILD

```diff
@@ -0,0 +1,270 @@
+# Contributor: Simon Frankenberger <simon-alpine@fraho.eu>
+# Maintainer: Simon Frankenberger <simon-alpine@fraho.eu>
+pkgname=openjfx11
+pkgver=11.0.3_p2
+_pkgver=${pkgver/_p/+}
+pkgrel=0
+pkgdesc="Open JavaFX 11"
+url="https://hg.openjdk.java.net/openjfx/11-dev/rt"
+arch="all !x86 !armhf !armv7" # openjdk11 is not yet available on 32 bit arches
+license="GPL-2.0 with Classpath"
```

Comment:
I don't think this conforms with SPDX?

Reply:
Don't know how to specify otherwise. It's the same licence as opendjk11:
https://pkgs.alpinelinux.org/package/edge/community/x86_64/openjdk11

Reply:
https://spdx.org/licenses/GPL-2.0-with-classpath-exception.html seems correct